test/workers/workarea/browse_option_index_product_test.rb

```diff
@@ -26,11 +26,11 @@ def test_clear
       Workarea::Search::Storefront.reset_indexes!
 
       foo_bar = create_product(id: 'FOO BAR')
-      IndexProduct.perform(create_product(id: 'FOO'))
-      IndexProduct.perform(create_product(id: 'BAR'))
-      IndexProduct.perform(foo_bar)
+      BulkIndexProducts.perform_by_models(create_product(id: 'FOO'))
```

Comment:
can't we just dump this test? It was testing the decoration of IndexProduct, which it doesn't do anymore.

Reply:
The bulk index tests weren't leaving me feeling like they had this well covered, and I thought it would better for upgrades to just leave it in place, testing the same thing.